.drone.yml

```diff
@@ -653,6 +653,60 @@ steps:
       - e2e/cloud-plugins-suite/azure-monitor.spec.ts
     repo:
     - grafana/grafana
+- commands:
+  - sleep 10s
```

Comment:
Q: added to resolve some kind of timing issues?

Reply:
Yes! It seems sometimes the server is not ready when playwright tests starts, so adding this just to be sure. Test tests run fast, so this shouldn't have an impact on the overall build time. 

Note that this file is generated from the lib.start file. I have added a comment [here](https://github.com/grafana/grafana/blob/0285aef332aeee7d9e505487a5dc833c82175402/scripts/drone/steps/lib.star#L862), but comments are being stripped out during file generation.